myna/core/components/component_class_lookup.py

```diff
@@ -43,7 +43,8 @@ def return_step_class(step_name):
         "cluster_supervoxel": ComponentClusterSupervoxel(),
         "microstructure_part": ComponentMicrostructurePart(),
         "microstructure_region": ComponentMicrostructureRegion(),
-        "rve": ComponentRVE(),
+        "rve_selection": ComponentRVE(),
+        "rve_part_center": ComponentCentroidRVE(),
```

Comment:
@gknapp1 I think it makes sense that if selecting off the part geometry, only an RVE in the center of the part is simulated - but to also support an option to select multiple regions of a part if needed. What is the current behavior of myna, does it allow multiple RVEs to be selected, and this is just adding a new default where if no RVEs are listed, it defaults to a component center?

Reply:
The current options are:

- manually define the XY locations and layers you want to include in any number of RVEs
- specify which parts & layers you want to investigate, but not a particular location. Then use a simulation to calculate G & V, apply the clustering algorithm, and automatically select an RVE from a representative region for each cluster

This change is adding a new option to specify the parts and layers you want to investigate and then have Myna define RVEs at the center of the part automatically based on the scan path data.

Reply:
Also, to clarify, this would mainly be used when launching Myna from Peregrine, where there isn't currently a way for a user to enter in a manual list of RVE locations. I imagine that manually entering RVE locations would still be the main way of setting things up outside of the Peregrine interface.

Reply:
So with the second option, it previously selected RVE locations automatically based on the clusters? And this is just saying that if you give Myna a part but no locations, it just defaults to a single RVE in the center of the simulated region?

Reply:
Pretty much, though you have to set up the workflow to explicitly use the clustering approach to RVE selection. I need to write some better documentation on how to do that.